examples/nlp/language_modeling/conf/megatron_gpt_config.yaml

```diff
@@ -69,9 +71,15 @@ model:
   # precision
   native_amp_init_scale: 4294967296 # 2 ** 32
   native_amp_growth_interval: 1000
+  hysteresis: 2 # Gradient scale hysteresis
   fp32_residual_connection: False # Move residual connections to fp32
   fp16_lm_cross_entropy: False # Move the cross entropy unreduced loss calculation for lm head to fp16
 
+  # Megatron O2-style half-precision
+  megatron_amp_o2: False # Enable O2-level automatic mixed precision using master parameters
```

Comment:
Small thing but can we change this to `megatron_amp_O2`?

Reply:
Done